dev/user.clj

```diff
@@ -0,0 +1,82 @@
+(ns dev.user
```

Comment:
The problem with `dev/user` is that it's usually reserved for operational things. Perhaps add to `dev/analysis.clj` or something similar?